item.rb

```diff
@@ -0,0 +1,17 @@
+class Item
+  attr_accessor :genre, :author, :source, :label, :publish_date, :archived
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def can_be_archived?
+    @publish_date > 10
+  end
```

Comment:
- [ ] Based on the requirement, there should be a test written for all public methods in classes. Kindly write tests for the `can_be_archived?` and `move_to_archive?` methods in the Item class since they are public in your classes. 👍